### PR TITLE
Adds mannitol and salbutamol to medical borgs hypospray

### PR DESCRIPTION
**Pull request by 1080pCat**
## What Does This PR Do
Adds two minor, yet overly useful chemicals to medborgs hypospray being mannitol and salbutamol.

## Why It's Good For The Game
It's always struck me as being very strange why medborg doesn't have these two basic chems, as they're majorly used within medbay. and adding these two chems can allow people playing medborg to go out and tag along with paramedics, and assist in stabilizing crew members.  

## Testing
I gave a skrell 500 oxy damage, changed the hypospray mode to salbutamol, injected 25 units, and verified that it injected the chemical with the med scanner, and watched as the oxy damage went down much faster than it usually does.
Same with mannitol, gave a skrell 50 brain damage, swapped the mode to mannitol, injected 25 units, and verified it injected the mannitol, and healed the brain damage.

## Changelog
:cl:
add: Adds salbutamol and mannitol to medborgs hypospray
/:cl:

### CODE REVIEW

**Comment by Miraviel** [comment]
As a reminder, silently downvoting PRs do little to nothing. If you disagree with something done here, explain why, so the contributor can either respond to it or alter their PR.

**Comment by S34NW** [comment]
Borgs are limited compared to carbon doctors in other ways, this is fine TBH.

File: code/modules/reagents/reagent_containers/borghydro.dm [hunk 1]
```diff
@@ -16,8 +16,8 @@
 	var/bypass_protection = 0 //If the hypospray can go through armor or thick material
 
 	var/list/datum/reagents/reagent_list = list()
-	var/list/reagent_ids = list("salglu_solution", "epinephrine", "spaceacillin", "charcoal", "hydrocodone")
-	//var/list/reagent_ids = list("salbutamol", "silver_sulfadiazine", "styptic_powder", "charcoal", "epinephrine", "spaceacillin", "hydrocodone")
+	var/list/reagent_ids = list("salglu_solution", "epinephrine", "spaceacillin", "charcoal", "hydrocodone", "mannitol", "salbutamol")
+	//var/list/reagent_ids = list("salbutamol", "silver_sulfadiazine", "styptic_powder", "charcoal", "epinephrine", "spaceacillin", "hydrocodone", "mannitol")
```
Comment (S34NW) [inline code/modules/reagents/reagent_containers/borghydro.dm:20]:
```suggestion
```

I'm not overly sure what this second, commented out, list is here for (it was there from before, not your fault) so you can remove it safely.
It's not actually doing anything.

**Comment by Gatchapod** [comment]
Mira shaming me into repeating myself, so I might as well go into details.

Cyborgs already are paramedics with built-in mobile OR. Brain damage is one of the very few things that keep them reliant on medbay and even that can be completely ignored with surgically fixing brains.
Salbutamol is almost completely useless given how quickly oxyloss can be made manageable by cyborgs. It will, however, let a single medical cyborg nullify lexorin during blob infestation. Lexorin blobs already are some of the weakest blobs and they do not need a nerf.

Furthermore, a medical cyborg telling you to take a pill instead of injecting you silently facilitates better roleplay in my opinion.

**Comment by matttheficus** [comment]
I echo @Gatchapod's concerns with the Lexorin blob counter. That, and making MedBorgs even MORE of a replacement for Doctors seems a bit too much for me.

**Comment by 1080pCat** [comment]
I cant really argue with the luxorin counter, aside from by the time a blob rolls around, people are likely going to have a self resp virus, and very likely perflu patches, so im on the fence on agreeing completely with this being a total counter, that and the moment the borg is caught by blob, its perma dead, so..

As with the 'replacing' organic doctors, this argument has always bothered me. i don't agree with it personally, medical cyborgs atleast from what it strikes me as, are meant to be able to stabilize crewmembers, and deal with the stuff doctors dont want to.

Something mira brought up while i was talking to them about medborgs that could alleviate the concerns, would be to give medborg three 'subtypes' of module, first responder, surgeon, and 'general care', however a entire rework of medborg may be a bit too much for a new coder, i'll look into it though.

**Comment by Gatchapod** [comment]
Perfluorodecalin patches have to be made and voluntarily taken by the player.
Self-respiration virus has to be made and voluntarily taken by the player.
Medical cyborg with salbutamol can be a roundstart thing and inject people without their input, especially those willing to fight but lacking mechanical knowledge to counter blob chemicals.
This creates a loop where they are winning, but they don't know why and they're unlikely to question why. People are more interested in why they lost than why they won. As such, they never learn. I consider this a bad thing.

It is not a total counter to lexorin blobs, but between IPCs being immune to oxyloss, salbutamol pills being readily available in oxygen lockers and popularity of self-respiration viruses, lexorin blobs ARE likely our weakest blobs and giving one cyborg ability to **force** multiple players to counter lexorin seems like a very bad change.

Regarding sub-modules, I like the idea on paper. Have you seen the medborg bloat? It has like 20 items and uses only a fraction of that. That, however, would probably require a decent design doc to look through and approve before you even attempt coding it.
Salbutamol concerns, however, would remain. To me, it just feels like punching lexorin blobs for no reason.

**Comment by SpringSkipper** [comment]
Salbutamol is available roundstart in med vendors and in oxygen first aid kits in various O2 closets in maintenance; it clearly isn't meant to be rare or else a lot of mistakes have been made in a lot of places. Managing oxygen damage is a very very basic aspect of emergency response and first aid in this game and it's something that medical borgs need to be able to do.

Intentionally gimping mediborgs because they're 'supposed to be weaker than doctors' is logic I simply can't follow and frankly feel is demeaning to the player behind the borg. Cyborgs are canonically supposed to be property and without right **in character**, but I feel like this has somehow bled over into how some people think their mechanical gameplay experience should be. Yes, cyborgs can be built, but they can't do anything without a player to pilot them. The idea that they should be fundamentally worse than the equivalent human job is flawed thinking in my opinion.

**Comment by procdrone** [comment]
give us the option to pick up pills. I wouldn't mind not being able to access Salbmutol or Mannitol from hypospray if I didn't have to find a doctor with hands to apply pills.

I agree with this PR, although if thats not acceptable, the above is a suggestion for solving this issue without giving the borg infinite chems.

**Review by S34NW — APPROVED** [review]

**Comment by 1080pCat** [comment]
> Salbutamol is available roundstart in med vendors and in oxygen first aid kits in various O2 closets in maintenance; it clearly isn't meant to be rare or else a lot of mistakes have been made in a lot of places. Managing oxygen damage is a very very basic aspect of emergency response and first aid in this game and it's something that medical borgs need to be able to do.

Not to mention medbots also give salbutamol, for far cheaper 'cost' than borg do..

**Review by SteelSlayer — APPROVED** [review]

**Pull request merged**